android/build.gradle

```diff
@@ -26,7 +26,7 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 20
```

Comment:
any reason for bumping this to 20?

Reply:
I've bumped the **app/build.gradle** to minSdk 20 because this was required for platform views. I got the following error when running the app on a device (simulator) with api level 19: 

```[ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: PlatformException(error, java.lang.IllegalStateException: Trying to use platform views with API 19, required API level is: 20```

For consistency I've also bumped this version number but this is not really required.

Reply:
sounds good!